src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/ActionCorrection.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class ActionCorrection {
+    private CorrectionActions action;

```

Comment:
Is there a reason why these are not just enum fields in the ActionPostHearingApplication class?

Reply:
This is a framework for future work, there will be more fields added